.github/workflows/main.yml

```diff
@@ -121,63 +115,35 @@ jobs:
       - name: Packaging
         run: |
           cp .buildkite_win/appxmanifest.xml kolibri-windows/
-          & "${Env:ProgramFiles(x86)}\Windows Kits\10\App Certification Kit\makeappx.exe" pack /d kolibri-windows /p endlesskeyapp.unsigned.appx
-
-      - name: Upload unsigned APPX
```

Comment:
Let's keep this stage uploading the unsigned package for all builds, so we can easily get it for checking if it pass validation with the store.

Reply:
ok